.prettierignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-.next
\ No newline at end of file
+package.json
```

Comment:
Should this include the lock file as well?

Reply:
Good call. I updated the commit.